cdc/model/changefeed.go

```diff
@@ -31,6 +33,50 @@ type ChangeFeedDetail struct {
 	// The ChangeFeed will exits until sync to timestamp TargetTs
 	TargetTs uint64          `json:"target-ts"`
 	Info     *ChangeFeedInfo `json:"-"`
+
+	filter              *filter.Filter
+	FilterCaseSensitive bool          `json:"filter-case-sensitive"`
+	FilterRules         *filter.Rules `json:"filter-rules"`
+}
+
+func (detail *ChangeFeedDetail) getFilter() *filter.Filter {
+	if detail.filter == nil {
```

Comment:
I recommend not to combine system level filter with user-defined `black-white-list`.
In current design, If users want to ignore only one non-system level schema such as `not_replica_db`, they have to write `ignore-dbs: ["INFORMATION_SCHEMA", "PERFORMANCE_SCHEMA", "mysql", "not_replica_db"]` ?

Reply:
Fixed.